cmd/podman/run.go

```diff
@@ -45,12 +46,15 @@ func runCmd(c *cli.Context) error {
 		return errors.Errorf("image name or ID is required")
 	}
 
-	imageName, _, data, err := imageData(c, runtime, c.Args()[0])
+	rtc := runtime.GetConfig()
+	newImage, err := runtime.ImageRuntime.New(c.Args()[0], rtc.SignaturePolicyPath, "", image.CreateStdoutWriter(), nil, image.SigningOptions{})
```

Comment:
Can we set SignaturePolicyPath to "" and have that use the default?

Reply:
i dont understand what you mean here.  why not use the configs signaturepolicypath ?

Reply:
The ImageRuntime is in the Runtime, the Runtime has the signature policy path, the runtime should be able to create the ImageRuntime and pass SignaturePolicyPath in

Reply:
fixed